tools/support_matrix/lib/support_matrix/ods_formatter.rb

```diff
@@ -0,0 +1,314 @@
+      property :cell,
+        "background-color" => "#ff9696",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "future_support", family: :cell do
+      property :cell,
+        "background-color" => "#ffffcc",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-weight" => "bold"
+    end
+
+    workbook.style "small-future_support", family: :cell do
+      property :cell,
+        "background-color" => "#ffffcc",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "supported", family: :cell do
+      property :cell,
+        "background-color" => "#00ff00",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-weight" => "bold"
+    end
+
+    workbook.style "small-supported", family: :cell do
+      property :cell,
+        "background-color" => "#00ff00",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "working", family: :cell do
+      property :cell,
+        "background-color" => "#83caff",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-weight" => "bold"
+    end
+
+    workbook.style "small-working", family: :cell do
+      property :cell,
+        "background-color" => "#83caff",
+        "border" => "0.002cm solid #000000"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "empty", family: :cell do
+      property :cell, "border" => "0.002cm solid #000000"
+    end
+
+    workbook.style "small", family: :cell do
+      property :cell, "border" => "0.002cm solid #000000"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "table-header", family: :cell do
+      property :cell, "background-color" => "#e6e6e6",
+                      "border" => "0.002cm solid #000000"
+    end
+
+    workbook.style "centered-small-table-header", family: :cell do
+      property :cell, "background-color" => "#e6e6e6",
+                      "border" => "0.002cm solid #000000"
+      property :paragraph, "text-align" => "center"
+      property :text, "font-size" => "8"
+    end
+
+    workbook.style "4cm-width", family: :column do
+      property :column, "column-width" => "4cm"
+    end
+
+    workbook.style "4cm-hight", family: :row do
+      property :row, "row-height" => "4cm"
+    end
+
+    workbook.style "3cm-hight", family: :row do
+      property :row, "row-height" => "3cm"
+    end
+
+    workbook.style "1.5cm-hight", family: :row do
+      property :row, "row-height" => "1.5cm"
+    end
+
+    workbook.style "vertical", family: :cell do
+      property :cell, "rotation-angle" => "90",
+                      "border" => "0.002cm solid #000000"
+    end
+
+    workbook.style "square", family: :column do
+      property :column, "column-width" => "0.7cm"
+    end
+  end
+
+  def integration_tests_sheet(matrix)
```

Comment:
Assignment Branch Condition size for integration_tests_sheet is too high. [94.35/15]